pkgs/development/python-modules/pyparsebluray/default.nix

```
@@ -0,0 +1,26 @@
+{
+  buildPythonPackage,
+  lib,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "pyparsebluray";
+  version = "0.1.4";
+
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
pyproject has a fallback for setuptools by adding it to build-system which is highöy prefered and compatible in almost all cases.